src/components/DaoCreator/GuardDetails.tsx

```diff
@@ -65,6 +74,55 @@ function GuardDetails() {
   const { t } = useTranslation(['daoCreate']);
   const votes = t('votes');
   const seconds = t('seconds');
+
+  useEffect(() => {
+    if (!isInitialized) {
+      if (governanceIsLoading || !safe || !governanceToken) return;
+
+      let totalVotes: number;
+      switch (type) {
+        case GovernanceTypes.GNOSIS_SAFE_USUL:
+          totalVotes = parseInt(
+            ethers.utils.formatUnits(governanceToken.totalSupply!, governanceToken.decimals)
```

Comment:
I guess I would rather see `governanceToken.totalSupply || '0'` here as opposed to the trailing ! (i might not be familiar enough on how that really works, as what if would the parameter value would be if it did happened to be undefined). I can't seem to reproduce it, but I was getting an error on this line reporting a value of "".

Reply:
```suggestion
            ethers.utils.formatUnits(governanceToken.totalSupply || '0', governanceToken.decimals)
```
I agree (trying out feature)

Reply:
good call, thanks for the easy commit button @Da-Colon !